incompatibility/incompatibility-report/incomp-scanning_test.go

```diff
@@ -45,16 +47,49 @@ func TestBasicSecurityIncompatibility(t *testing.T) {
 	path1 := "../../generator/testfiles/other.yaml"
 	path2 := "../../examples/petstore/petstore.yaml"
 
-	var serversTest = []struct {
+	var securityTest = []struct {
 		path           string
 		expectSecurity bool
 	}{
 		{path1, false},
 		{path2, true},
 	}
-	for _, tt := range serversTest {
+	for _, tt := range securityTest {
 		if incompatibilityCheck(generateDoc(t, tt.path), IncompatibiltiyClassification_Security) != tt.expectSecurity {
```

Comment:
I believe it's best practice to use t.Run() so you can name and identify the specific test which has failed.

Thoughts?

Reply:
now using t.run